pkg/config/config.go

```diff
@@ -0,0 +1,60 @@
+package config
+
+import (
+	networkaddons "github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/v1alpha1"
+	hcov1alpha1 "github.com/kubevirt/hyperconverged-cluster-operator/pkg/apis/hco/v1alpha1"
+	cdi "kubevirt.io/containerized-data-importer/pkg/apis/core/v1alpha1"
+	kubevirt "kubevirt.io/kubevirt/pkg/api/v1"
+
+	v1 "k8s.io/api/core/v1"
+)
+
+var (
+	DefaultVersion           = "2.0"
+	DefaultContainerRegistry = "quay.io/kubevirt"
+	DefaultImagePullPolicy   = v1.PullIfNotPresent
+)
+
+// NewHCOSpec - returns an HCOSpec object populated by default variables
+func NewHCOSpec(HcoVersion string, HcoContainerRegistry string, HcoImagePullPolicy v1.PullPolicy) *hcov1alpha1.HyperConvergedSpec {
+	if HcoVersion == "" {
+		HcoVersion = DefaultVersion
+	}
+	if HcoContainerRegistry == "" {
+		HcoContainerRegistry = DefaultContainerRegistry
+	}
+	if string(HcoImagePullPolicy) == "" {
+		HcoImagePullPolicy = DefaultImagePullPolicy
+	}
+
+	return &hcov1alpha1.HyperConvergedSpec{
+		Version:           HcoVersion,
+		ContainerRegistry: HcoContainerRegistry,
+		ImagePullPolicy:   HcoImagePullPolicy,
+	}
+}
+
+// NewKubeVirtSpec - returns a KubeVirtSpec object populated by HCO variables
+func NewKubeVirtSpec(hcoSpec *hcov1alpha1.HyperConvergedSpec) kubevirt.KubeVirtSpec {
+	return kubevirt.KubeVirtSpec{
+		ImageTag:        hcoSpec.Version,
```

Comment:
So KubeVirt has to be able to respect the HCO versions? Out of the box, if the HCO version were `0.1.0` wouldn't you end up with some old  version of KubeVirt...or worse if the version is `1.0.0` you would be trying to install a KubeVirt version that doesn't exist.

Reply:
Yes.  See https://github.com/kubevirt/hyperconverged-cluster-operator/pull/22#discussion_r273214077.  Downstream, this will work.  Upstream, this is a problem which we are going to have a follow up discussion on.